runtime/cfdumper/romdump.c

```diff
@@ -698,14 +700,14 @@ parseROMQuery(J9Pool *allocator, const char *query)
 
 		if ('[' == *query) {
 			query++;
-			if (!isdigit(*query)) {
+			if (!__isdigit_a(*query)) {
```

Comment:
Is this a function on z/OS? The code is common to all platforms and needs to compile on all of them.

Reply:
This should instead use `OMR_ISDIGIT()` once https://github.com/eclipse-omr/omr/pull/7711 is merged.